spot_driver/spot_driver/spot_ros2.py

```diff
@@ -1411,19 +1459,14 @@ def handle_manipulation_command(self, goal_handle: ServerGoalHandle) -> Manipula
             feedback_msg = Manipulation.Feedback(feedback=feedback)
             goal_handle.publish_feedback(feedback_msg)
 
-            goal_complete = feedback is not None and (
-                feedback.current_state == manipulation_api_pb2.MANIP_STATE_GRASP_SUCCEEDED
-                or feedback.current_state == manipulation_api_pb2.MANIP_STATE_GRASP_FAILED
-            )
-
             time.sleep(0.1)  # don't use rate here because we're already in a single thread
 
         # publish a final feedback
         result = Manipulation.Result()
         if feedback is not None:
             goal_handle.publish_feedback(feedback_msg)
             result.result = feedback
-        result.success = self._goal_complete(feedback) == GoalResponse.SUCCESS
+        result.success = self._manipulation_goal_complete(feedback) == GoalResponse.SUCCESS
```

Comment:
Should these still be returning this custom `GoalResponse` type?

Reply:
I used the same one for consistency